hw/dv/dpi/usbdpi/usbdpi.c

```diff
@@ -647,7 +650,7 @@ void getTestConfig(usbdpi_ctx_t *ctx, uint16_t desc_len) {
             const uint8_t test_sig_tail[] = {0x1fu, 0x0cu, 0x75u, 0xe7u};
             if (!memcmp(dp, test_sig_head, 4) && 0x10 == get_le16(&dp[4]) &&
                 !memcmp(&dp[12], test_sig_tail, 4)) {
-              ctx->test_number = get_le16(&dp[6]);
+              ctx->test_number = (usb_testutils_test_number_t)get_le16(&dp[6]);
```

Comment:
Our coding style requires that there be no space, and it would be unusual to introduce a space between a type cast and expression. CI/clang_format_fix would reject it anyway.